borsh-derive/src/internals/schema/enums/snapshots/borsh_discriminant_true.snap

```diff
@@ -42,23 +42,17 @@ impl borsh::BorshSchema for X {
         <XD as borsh::BorshSchema>::add_definitions_recursively(definitions);
         <XE as borsh::BorshSchema>::add_definitions_recursively(definitions);
         <XF as borsh::BorshSchema>::add_definitions_recursively(definitions);
-        let discriminant_0: u8 = 0;
-        let discriminant_1: u8 = 20;
-        let discriminant_2: u8 = 20 + 1;
-        let discriminant_3: u8 = 20 + 1 + 1;
-        let discriminant_4: u8 = 10;
-        let discriminant_5: u8 = 10 + 1;
         let definition = borsh::schema::Definition::Enum {
             tag_width: 1,
             variants: borsh::__private::maybestd::vec![
-                (discriminant_0 as i64, "A".to_string(), < XA as borsh::BorshSchema >
```

Comment:
the pr would be simpler to review if it didn't contain `to_string` -> `into` changes, which are a little bit off from its subject